rfcs/react-components/convergence/global-contex-registration.md

```diff
@@ -0,0 +1,148 @@
+# RFC: Global context registration
+
+@ling1726
+
+## Summary
+
+This RFC proposes that all Fluent UI contexts be registered in the global scope when created. Global scope means
+`window` for browser environments or the `global` object for nodejs environments when using SSR.
+
+## Problem statement
+
+Many of our partners have complex dependency trees. The complexity in dependency management means that duplicate
+packages can exist in `node_modules` folders. Although duplicate depedencies should generally be avoided, this still
+happens quite often due to these factors:
```

Comment:
I like that idea. I am just not optimistic that it will be enough. From what I've seen partners will currently tolerate multiple versions of Fluent in the library anyway.